vtex.env

```diff
@@ -1,6 +1,7 @@
 # configures Gatsby
 GATSBY_CPU_COUNT=4
 ENABLE_GATSBY_REFRESH_ENDPOINT=true
+GATSBY_QUERY_ON_DEMAND_LOADING_INDICATOR=false
```

Comment:
Is this being used anywhere?

![CleanShot 2022-04-04 at 16 10 16](https://user-images.githubusercontent.com/381395/161615511-401f2954-e9fa-4da0-b9f5-dd0add9eed97.png)

```suggestion
```

Reply:
Yes, this is used by Gatsby internally. https://github.com/gatsbyjs/gatsby/blob/ee47e27c843910bf08887f270b831bbaeb6ab538/packages/gatsby/cache-dir/app.js#L163